package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opuscapita/react-crudeditor",
-  "version": "1.0.3",
+  "version": "1.0.3-issue-262-SNAPSHOT",
```

Comment:
Version change is not needed as the package is not deployed during a build process.